tests/badge.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { test, expect } from '@playwright/test';
+import config from '../tailwind.config.cjs'
+
+const hexToRGB = (hex) => {
+  let alpha = false;
+  let h = hex.slice(hex.startsWith('#') ? 1 : 0);
+  if (h.length === 3) h = [...h].map(x => x + x).join('');
+  else if (h.length === 8) alpha = true;
+  h = Number.parseInt(h, 16);
+  return (
+    `rgb${ 
+      alpha ? 'a' : '' 
+    }(${ 
+      h >>> (alpha ? 24 : 16) 
+    }, ${ 
+      (h & (alpha ? 0x00_FF_00_00 : 0x00_FF_00)) >>> (alpha ? 16 : 8) 
+    }, ${ 
+      (h & (alpha ? 0x00_00_FF_00 : 0x00_00_FF)) >>> (alpha ? 8 : 0) 
+    }${alpha ? `, ${h & 0x00_00_00_FF}` : '' 
+    })`
+  );
+};
+
+
+test('Badge E2E Tests', async ({ page }) => {
+  await page.goto('/test.html');
+
+  // Gray Badge Test
+
+  const grayBadge = page.getByText('Inactive')
+  
+  await expect(grayBadge).toBeVisible()
+  await expect(grayBadge).toHaveText("Inactive")
+  
+
+  await expect(grayBadge).toHaveCSS("background-color", hexToRGB(config.theme.extend.colors['gray/200']))
```

Comment:
I believe that playwright css color assertions only support rgb so I needed to convert from hex to rgb to get this to work.